docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,518 @@ aliases:
 
 # Brigade Gateways
 
-This guide explains how gateways work, and provides guidance for creating your own
-gateway.
+This guide explains how gateways work, and provides guidance for creating your
+own gateway.
```

Comment:
Total nit: "gateways" (plural) might read better here. _Total_ nit. Dealer's choice.

Reply:
Thanks, just left it at `... for creating your own.`

Reply:
I like that ever better. Nice.